site/app/controllers/pdf/PDFController.php

```diff
@@ -57,7 +54,7 @@ public function showStudentPDF($gradeable_id, $filename, $path, $grader = null)
             "gradeable_id" => $gradeable_id,
             "id" => $id,
             "file_name" => $filename,
-            "file_path" => urldecode($path),
+            "file_path" => urldecode($decoded_path),
```

Comment:
why are we double `urldecode` on this path? Not sure if there's an actual harm in this, but also should avoid if we do not have to.

Reply:
Removed the double decode